client/packages/invoices/src/Prescriptions/DetailView/AppBarButton.tsx

```diff
@@ -60,10 +60,9 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   return (
     <AppBarButtonsPortal>
       <Grid container gap={1}>
-        <ButtonWithIcon
+        <AddButton
           disabled={isDisabled}
           label={t('button.add-item')}
-          Icon={<PlusCircleIcon />}
           onClick={() => onAddItem()}
```

Comment:
Not related to this issue, but just noticed -- there's no need to make an anonymous function here, can just set `onClick={onAddItem}` -- not sure if it's worth changing 🤷‍♂️
 
(And the `onViewHistory()` just below too ;) )

Reply:
`onAddItem` you can, bc the onClick has no parameter
`onViewHistory` though you can't. That button's `onClick` is the standard and has an optional parameter which differs from the `onViewHistory` optional. you get that pattern throughout the codebase, which is why there are so many of these anonymous functions which look redundant.